chrome_manifest_v3/class_app.js

```
@@ -0,0 +1,869 @@
+    return body;
+  }
+
+  /**
+   * Determine luminance of a color so we can augment with darker/lighter background
+   */
+  luminance(color) {
+    const bkColorLight = 'lightGray'; // or white
+    const bkColorDark = 'darkGray'; // 'gray' is even darker
+    let bkColorReturn = bkColorLight;
+
+    const re = new RegExp('rgb\\D+(\\d+)\\D+(\\d+)\\D+(\\d+)');
```

Biome (1.9.4)

[error] Use a regular expression literal instead of the RegExp constructor.

Regular expression literals avoid some escaping required in a string literal, and are easier to analyze statically.
Safe fix: Use a literal notation instead.


(lint/complexity/useRegexLiterals)